src/ocamlorg_frontend/pages/about.eml

```diff
@@ -50,243 +50,168 @@ Layout.render
-                        call-by-value functional language, as well as inductive types and pattern matching. With Core ML
-                        as a launchpad, several new languages developed including Caml, the forerunner to OCaml.
+                        All of this makes OCaml a great choice for programmers who want to step up to a better
+                        programming language, and at the same time get practical work done.
                     </p>
+                    <h2 id="history">A Brief History</h2>
                     <p>
-                        The ‘Cam’ in OCaml comes from ML running on the Categorical Abstract Machine, which was notable
-                        for being one of the first formalisations of function closures. Caml was essentially Core ML but
-                        with provisions for embedded languages such as parser, quotations, and anti-quotations. It was
-                        developed alongside the Coq proof assistant, as Coq’s implementation language. Caml had a lot of
-                        strengths and promise but was ultimately rather inefficient.
+                        OCaml was written in 1996 by Xavier Leroy, Jérôme Vouillon, Damien Doligez, and Didier Rémy at
+                        INRIA in France. It was inspired by a long line of research into ML, starting in the 1960s, and
+                        continues to have deep links to the academic community.
                     </p>
                     <p>
-                        In an effort to understand why Caml was so ineffective, Xavier Leroy and D. Doligez conducted
-                        the Zinc Experiment. This deep dive into Caml eventually lead to Caml Light, which was more
-                        practically usable than its predecessor. Caml Light added and expanded on some new features,
-                        such as separate compilation and linking and a toplevel interactive REPL. It was also
-                        bootstrapped as well as impressively available on multiple platforms: Unix, Mac OS, and MS-DOS.
-                        Caml Light became popular as a teaching language and appeared in several books.
+                        ML was originally the meta language of the LCF (Logic for Computable Functions) proof assistant,
+                        released by Robin Milner in 1972 (at Stanford, and later at Cambridge). ML was turned into a
+                        compiler in order to make it easier to use LCF on different machines, and it was gradually
+                        turned into a full-fledged system of its own by the 1980s.
                     </p>
                     <p>
-                        In the 1990’s, objective orientation became incredibly popular, and non object-oriented
-                        languages were seen as a thing of the past. Adapting Caml to this new way of the world, D. Rémy
```

Comment:
removing this once again makes the text less compelling to read in my opinion, I think the audience doesn't just want to know what happened but WHY it happened